### PR TITLE
[WIPTEST]test for cloud provision with tag

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
Add new test: provision with tag assigment for cloud providers

{{pytest: -v -k test_cloud_provision_dialog_tag --long-running --use-provider complete}}

### CODE REVIEW

File: cfme/tests/cloud/test_provisioning.py [hunk 1]
```diff
@@ -90,41 +90,52 @@ def testing_instance(request, setup_provider, provider, provisioning, vm_name):
...
+        logger.info('Waiting for cfme provision request for vm %s', instance.name)
+        row_description = 'Provision from [{}] to [{}]'.format(image, instance.name)
+        cells = {'Description': row_description}
+        try:
+            row, __ = wait_for(requests.wait_for_request, [cells],
+                               fail_func=requests.reload, num_sec=1500, delay=20)
+        except Exception as e:
+            requests.debug_requests()
+            raise e
+        assert normalize_text(row.status.text) == 'ok' and \
```
Comment (quarckster) [inline cfme/tests/cloud/test_provisioning.py]: Please use braces instead of `\`

**Comment by dajoRH** [comment]
Lint report for commit 74810df8dd64547e4e1e080aa03b0e16cc59e4df:

`cfme/tests/cloud/test_provisioning.py`:
- :bangbang: Line 107:16: **E127** *continuation line over-indented for visual indent*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/cloud/test_provisioning.py (hunk 1 above)
Comment (rlbabyuk) [inline cfme/tests/cloud/test_provisioning.py]: Thanks, updated

**Comment by dajoRH** [comment]
Lint report for commit 9cf225ece47f106aa7c875513faf9c5813ba13e3:

`cfme/tests/cloud/test_provisioning.py`:
- :red_circle: Line 106:9: **F999** *assertion is always true, perhaps remove parentheses?*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Pull request closed without merging**